docs/DeveloperGuide.md

```diff
@@ -510,6 +529,50 @@ _{More to be added}_
 
     Use case ends.
 
+
+**Use case: Add a contact as favourite**
+
+**MSS**
+
+1. User requests to add a contact as favourite
+2. System marks the contact as favourite
+
+   Use case ends.
+
+**Extensions**
```

Comment:
Perhaps it would be good to include the extension where system detects some contacts are already favourited, and reacts by issuing a warning